config

```diff
@@ -500,12 +503,7 @@ case "$GUESSOS" in
 	if [ "$ISA64" = "1" -a -z "$KERNEL_BITS" ]; then
 	    echo "WARNING! If you wish to build 64-bit library, then you have to"
 	    echo "         invoke 'KERNEL_BITS=64 $THERE/config $options'."
-	    if [ "$DRYRUN" = "false" -a -t 1 ]; then
-	      echo "         You have about 5 seconds to press Ctrl-C to abort."
-	      # The stty technique used elsewhere doesn't work on
-	      # MacOS. At least, right now on this Mac.
-	      sleep 5
-	    fi
+	    sleep $SLEEPTIME
```

Comment:
This would be even more annoying. I mean dry-run shouldn't sleep. But why "stty technique" at all? Trouble is that output from config is so big that message about alternatives disappears, and without delay it disappears too quickly for user to even realize that there are alternatives. And once again, with "stty technique" you have both delay and control over how long you wait. I mean if you know what message is you simply press return.

Reply:
Okay, it's more complicated than I thought. Closing this and walking away :)